packages/slate-editor/src/extensions/table/components/elements/TableCellElement.tsx

```diff
@@ -1,45 +0,0 @@
-import type { TableCellNode } from '@prezly/slate-types';
-import React from 'react';
-import type { RenderElementProps } from 'slate-react';
-
-import styles from './elements.module.scss';
-import { TableContext } from './TableContext';
-
-interface Props extends RenderElementProps {
-    element: TableCellNode;
-}
-
-export function TableCellElement({ attributes, element, children }: Props) {
-    const ctx = React.useContext(TableContext);
-
-    const isHeaderCell = React.useMemo(() => {
-        const shouldCheckFirstRow = ctx.table.header?.some((h) => h === 'first_row');
-
-        if (shouldCheckFirstRow) {
-            const isCurrentCellInFirstRow = ctx.table.children?.[0].children?.some(
-                (cell) => cell === element,
-            );
-
-            if (isCurrentCellInFirstRow) {
-                return isCurrentCellInFirstRow;
-            }
-        }
-
-        return false;
-    }, [ctx.table, element]);
```

Comment:
This has been simplified to this: https://github.com/prezly/slate/pull/224/files#diff-052191ca2a1769b9020c25d8bed3e2a138d7e28ed977e5fba70f0b748f91fbe0R20-R25